hnn_core/network.py

```diff
@@ -487,7 +487,10 @@ def add_evoked_drive(self, name, *, mu, sigma, numspikes, location,
         numspikes : int
             Number of spikes at each target cell
         location : str
-            Target location of synapses ('distal' or 'proximal')
+            Target location of synapses. Must be an element of
+            `Cell.sect_loc` such as 'proximal' or 'distal', which defines a
+            group of sections, or an existing section such as 'soma' or
+            'apical_tuft' (defined in `Cell.sections` for all targeted cells)
```

Comment:
+1 for this. Eventually, it'd be nice to allow users to specify which cell sections apply to a given `sect_loc`. I can imagine a few different network changes I'd like to implement for my own project where the location (i.e., 'proximal' or 'distal') isn't necessarily intuitive.